recipes/amap/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.2" %}
+
+package:
+  name: amap
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/mes5k/amap-align/archive/600fc29fb9d0e8a6abf797c173c7a416ab99c541.zip
+  sha256: 57391561dfa0ec0e291a11f54177cfcf17baa47be065f0cc688e4bb98de792b4
+  patches:
+    - patch.file 
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    
+  run:
+    - blast
+
+test:
+  commands:
+    - amap -gui
+
+about:
+  home: "https://web.archive.org/web/20060902044446/http://bio.math.berkeley.edu/amap/"
+  license: "GPL"
```

Comment:
Do you know which version of GPL? Can you add a license_file?

Thanks!

Reply:
The file is in /license/License.txt on the zip file

should I write something like : ??
```
about:
  home: https://web.archive.org/web/20060902044446/http://bio.math.berkeley.edu/amap/
  license: GPL-2
  license_file: /license/License.txt
  summary: JAMAP is a multiple sequence alignment program based on sequence annealing
  dev_url: https://github.com/mes5k/amap-align
```

Reply:
hey @bgruening I hope it works now. 
But if it's not the best way to add the license just let me know 😄

Reply:
It looks good with the most recent changes. I'll merge once it passes testing.